src/stories/Numbers.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { Numbers } from '../components/Numbers';
+
+export default {
+  title: 'Numbers',
+  component: Numbers,
+  decorators: [withKnobs]
+};
+
+const data = {
+  _id: '5ea8c869d7c5de203463948e',
```

Comment:
try to use id instead of _id in your data in story and also in the main component

Reply:
Thanks : )